examples/rds-getengineversion/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as aws from "@pulumi/aws";
+
+export const vs = aws.rds.getEngineVersionOutput({engine: "postgres"});
```

Comment:
Very tangential - but should we be defining new tests in YAML these days? @t0yv0 @mjeffryes

Reply:
Yeah I think the slight preference is for YAML, but the cross-testing advantages of having it in YAML are not yet realized anywhere I think.